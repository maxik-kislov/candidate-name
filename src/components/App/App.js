import React from 'react';
import Feed from '../Feed/Feed';
import './App.scss';

const App = () => (
  <div className="app">
    <Feed />
  </div>
);

export default App;
