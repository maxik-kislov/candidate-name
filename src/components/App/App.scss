.app {
  margin: 0 auto;
  max-width: 1100px;
  padding: 0 8px;
  margin-top: 40px;
  font-family: Ubuntu, Open Sans, Segoe UI, Tahoma, sans-serif;
}

@media (min-width: 768px) {
  .app {
    max-width: 750px;
    padding: 0 16px;
  }
}
@media (min-width: 992px) {
  .app {
    max-width: 970px;
  }
}
@media (min-width: 1200px) {
  .app {
    max-width: 1170px;
  }
}
