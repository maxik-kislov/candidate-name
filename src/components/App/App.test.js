import setupTests from '../../setupTests';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import App from './App';

const wrapper = shallow(<App />);

describe('App', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should render correctly', () => {
    expect(wrapper.getElements()).toMatchSnapshot();
  });

  it('should contain an element with a class .app', () => {
    expect(wrapper.find('.app').exists()).toBeTruthy();
  });

  it('should contain a Feed component', () => {
    expect(wrapper.find('Feed').exists()).toBeTruthy();
  });
});
