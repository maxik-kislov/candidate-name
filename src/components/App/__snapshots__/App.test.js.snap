// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
Array [
  <div
    className="app"
  >
    <Feed />
  </div>,
]
`;
